content/videos/noc/3-angles/4-polar-coordinates/index.json

```diff
@@ -10,31 +10,31 @@
     {
       "title": "Basic Polar Coordinates",
       "description": "A simple example of using polar coordinates in p5.js",
-      "image": "image1.png",
+      "image": "basic.jpg",
       "urls": {
         "p5": "https://editor.p5js.org/codingtrain/sketches/DHetsOfgz"
       }
     },
     {
       "title": "Spiral",
       "description": "Using polar coordinates to draw a spiral",
-      "image": "image1.png",
+      "image": "spiral.jpg",
       "urls": {
         "p5": "https://editor.p5js.org/codingtrain/sketches/BnJ_4U4cr"
       }
     },
     {
       "title": "Random Trail",
       "description": "Drawing a glowing trail with polar coordinates",
-      "image": "image1.png",
+      "image": "trail.jpg",
       "urls": {
         "p5": "https://editor.p5js.org/codingtrain/sketches/WqK-qD1vU"
       }
     },
     {
       "title": "Circular Shape",
       "description": "Drawing a random circular shape with polar coordinates",
-      "image": "image1.png",
+      "image": "circular.jpg",
```

Comment:
```suggestion
      "image": "circlular.jpg",
```

Reply:
The image file had a typo in it so as a bandaid / quick fix I am just changing the name in the source.